core/src/main/java/io/questdb/cutlass/line/http/LineHttpSender.java

```diff
@@ -444,6 +474,7 @@ private HttpClient.Request newRequest() {
 
     private void throwOnHttpErrorResponse(DirectUtf8Sequence statusCode, HttpClient.ResponseHeaders response) {
         // be ready for next request
+        flushAfterNanos = Long.MAX_VALUE;
```

Comment:
I'm not sure I follow this bit.
What is the intended behaviour after a failed HTTP request?

Reply:
The intent: If there is a failure then we propagate an exception to a caller and a subsequent `at()` behaves as if it was a newly created `Sender` instance: A first added row starts a timer and next `at()` flushes if a flush policy says so. Does it make sense?

Reply:
@amunra does it make sense to you?